docs/reference/ccr/requirements.asciidoc

```diff
@@ -41,58 +41,3 @@ to a follower the following process will fail due to incomplete history on the l
 The default value is `12h`.
 
 For more information about index settings, see {ref}/index-modules.html[Index modules].
-
-
-[[ccr-overview-beats]]
```

Comment:
It looks like the Beats and APM docs link to this section. Can we add a redirect to [`redirects.asciidoc`](https://github.com/elastic/elasticsearch/blob/master/docs/reference/redirects.asciidoc) so these links don't break?

Reply:
That note (elastic/beats#9427) can be removed from the docs, as it no longer applies.

Reply:
I opened elastic/beats#15506 to remove the note in the Beats/APM docs. The docs CI tests will continue to fail for this PR until that's merged or we add a redirect.

Reply:
@dnhatn  elastic/beats#15506 is now merged. If you update this branch, the docs CI test should pick up the changes in Beats and pass.